P/p3a/build_tarballs.jl

```diff
@@ -0,0 +1,68 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "p3a"
+version = v"1.0.2"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/sandialabs/p3a.git", "271cd1d77fbc850aa4ddfb015b59176926670b3b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/p3a/
+mkdir build && cd build
+
+CMAKE_FLAGS=()
+
+#help find mpi on mingw, subset of https://github.com/JuliaPackaging/Yggdrasil/blob/b4fdb545c3954cff218051d7520c7418991d3416/T/TauDEM/build_tarballs.jl#L28-L53
```

Comment:
https://github.com/JuliaPackaging/Yggdrasil/blob/2fc2e9a55180b90ab96f20e582bdde2913c6a940/A/ADIOS2/build_tarballs.jl#L31-L54
might help?

Reply:
I ended up grabbing the block from [amrex](https://github.com/JuliaPackaging/Yggdrasil/blob/master/A/AMReX/build_tarballs.jl#L28-L35 and it seems to make everything happy?

Reply:
Yup, it's substantially the same